Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -276,13 +312,15 @@ - (void)toxAV:(OCTToxAV *)toxAV receiveCallAudioEnabled:(BOOL)audio videoEnabled
 
 - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friendNumber:(OCTToxFriendNumber)friendNumber
 {
-    OCTCall *call = [self getOrCreateCallWithFriendNumber:friendNumber];
+    OCTCall *call = [self getCurrentCallForFriendNumber:friendNumber];
 
     if ((state & OCTToxAVFriendCallStateFinished) || (state & OCTToxAVFriendCallStateError)) {
 
         [self addMessageCall:call];
```

Comment:
Can you please rename `addMessageCall:` to `addMessageAndDeleteCall:`. It is confusing now.